app/assets/stylesheets/buttons.scss

```diff
@@ -261,11 +266,14 @@
     }
 
     // Make child links fill the whole clickable area
-    > a {
+    > a,
+      .dropdown-items-link {
       display: flex;
       flex-grow: 1;
       margin: -$default-padding;
       padding: $default-padding;
+      text-align: left;
+      font-size: 12px;
```

Comment:
pourquoi on viendrait a forcer la taille de la font ? ne peut-on pas laisser le DSFR s'occuper de la taille des polices ? (ps: pas certains que ca soit top niveau a11y les font en px: https://medium.com/weekly-webtips/to-increase-a11y-use-rem-instead-of-px-96ee9d9dc9ad)

Reply:
j'ai retiré ce style concernant la font. Ça n'est plus d'actualité j'imagine depuis que j'utilise le `h4`